example/python/monogenicImageTest.py

```diff
@@ -0,0 +1,190 @@
+        display_ysize = int(round(ysize * scaling_factor))
+
+        # Ensure minimum size if needed (optional)
+        # if display_xsize < 100: display_xsize = 100
+        # if display_ysize < 100: display_ysize = 100
+
+        print(f"Images scaled down by factor: {scaling_factor:.2f} to fit on screen.")
+        print(f"New display size: {display_xsize}x{display_ysize}")
+
+    else:
+        print(f"Images displayed at original size: {display_xsize}x{display_ysize}")
+
+    # Offset for the next window based on the calculated display size
+    x_offset = display_xsize + WINDOW_SPACING
+
+    # Create display windows and position them side by side
+    window_names = ["Even", "Odd Y", "Odd X", "Feature Symmetry", "Feature Asymmetry"]
+    windows = {}
+
+    # Create a dummy image for initial display to help with window positioning
+    # Resize the input image to the calculated display size
+    initial_display_img = cv2.resize(input_image_np, (display_xsize, display_ysize))
+    # Convert to 3 channels if needed for color display, though grayscale is fine
+    # initial_display_img = cv2.cvtColor(initial_display_img, cv2.COLOR_GRAY2BGR)
+
+
+    for i, name in enumerate(window_names):
+        cv2.namedWindow(name, cv2.WINDOW_AUTOSIZE)
+        # Display something initially to ensure the window is created before moving
+        cv2.imshow(name, initial_display_img)
+        cv2.moveWindow(name, start_x + i * x_offset, start_y)
+        windows[name] = True # Keep track of created windows
+
+    # The first step to using the monogenicProcessor is to initialise
+    # a monogenic processor object.
+    # We'll use a wavelength of 50 pixels as in the C++ example
+    wavelength = 50.0
+
+    try:
+        mgFilts = pymonogenic.MonogenicProcessor(ysize, xsize, wavelength)
+        print("MonogenicProcessor object created.")
+    except Exception as e:
+        print(f"Error creating MonogenicProcessor: {e}")
+        # Destroy windows before exiting
+        for name in window_names:
+            if name in windows:
+                cv2.destroyWindow(name)
+        sys.exit(1)
+
+
+    # --- Start Profiling ---
+    start_time = time.time()
+
+    # Calculate the monogenic signal for the input image
+    # The wrapped method should handle the NumPy to cv::Mat conversion
+    try:
+        mgFilts.findMonogenicSignal(input_image_np)
+        # Now we can access the odd and even components and derived measures
+        # Assuming these methods are wrapped to return NumPy arrays
+        fa_np = mgFilts.getFeatureAsymmetry()
+        # --- End Profiling ---
+        end_time = time.time()
+        elapsed_time = end_time - start_time
+
+        # Print the elapsed time
+        print(f"Monogenic signal and feature calculation time: {elapsed_time:.4f} seconds")
+
+        even_np = mgFilts.getEvenFilt()
+        oddy_np, oddx_np = mgFilts.getOddFiltCartesian() # Assuming it returns a tuple
```

Comment:
```suggestion
        oddy_np, oddx_np = mgFilts.getOddFiltCartesian()
```

This is a potentially confusing comment for someone reading it. Why do we need to assume anything?

Reply:
agreed - hangover from deving things not required